### PR TITLE
update grafana node view

### PR DESCRIPTION
**Pull request by YitongFeng**
Have done the test and deployed at big gpu cluster. See:
Cluster view: http://10.151.40.179:3000/dashboard/db/pai_clusterview?refresh=30s&orgId=1 
Node view: http://10.151.40.179:3000/dashboard/db/pai_nodeview?refresh=30s&orgId=1 

### Update log:
- Add GPU usage gauge panel in cluster view
- Redesign layout of cluster view
- Set default time range and refresh time (see top right corner)
- Add drop-down box of node view (see top left corner of it)
- Remove graph titles' IP prefix
- Change GPU count to int

Other advice for revision welcomed.

### CODE REVIEW

File: service-deployment/src/grafana/pai-nodeview-dashboard.json.template [hunk 1]
```diff
@@ -493,7 +401,7 @@
           "thresholds": [],
           "timeFrom": null,
           "timeShift": null,
-          "title": "$node Disk Space Used",
+          "title": "Disk Space Used",
```
Comment (hwuu) [inline service-deployment/src/grafana/pai-nodeview-dashboard.json.template]: Is it really 'disk space used'? I found the page calls promethus api to get disk read/write rates.

Comment (YitongFeng) [reply service-deployment/src/grafana/pai-nodeview-dashboard.json.template]: Changed title into "Disk"

File: service-deployment/src/grafana/pai-clusterview-dashboard.json.template [hunk 2]
```diff
@@ -42,13 +43,14 @@
...
           },
+          "height": "312px",
           "id": 9,
           "interval": null,
           "links": [
             {
               "dashboard": "PAI Cluster Node List",
               "type": "absolute",
-              "url": "{{ clusterconfig['clusterinfo']['grafanainfo']['grafana_url'] }}:{{ clusterconfig['clusterinfo']['grafanainfo']['grafana_port'] }}/dashboard/db/pai_clusterview"
+              "url": "http://10.151.40.179:3000/dashboard/db/pai_clusterview"
```
Comment (sterowang) [inline service-deployment/src/grafana/pai-clusterview-dashboard.json.template]: I guess this change is for testing purpose on devbox? Should change back to value from clsuterconfig.

File: service-deployment/src/grafana/pai-nodeview-dashboard.json.template [hunk 3]
```diff
@@ -721,22 +629,26 @@
...
                 "rgba(50, 172, 45, 0.97)"
               ],
               "dateFormat": "YYYY-MM-DD HH:mm:ss",
-              "decimals": 2,
+              "decimals": 0,
               "link": true,
-              "linkTooltip": "",
-              "linkUrl": "{{ clusterconfig['clusterinfo']['grafanainfo']['grafana_url'] }}:{{ clusterconfig['clusterinfo']['grafanainfo']['grafana_port'] }}/dashboard/script/gpu.js?orgId=1&host=$__cell_2&rows=$__cell_4",
+              "linkTooltip": "Click to view usage  of each GPU",
+              "linkUrl": "http://10.151.40.179:3000/dashboard/script/gpu.js?orgId=1&host=$__cell_2&rows=$__cell_4",
```
Comment (sterowang) [inline service-deployment/src/grafana/pai-nodeview-dashboard.json.template]: The same for hardcoded address.

File: service-deployment/src/grafana/pai-nodeview-dashboard.json.template [hunk 4]
```diff
@@ -848,14 +748,14 @@
         "allFormat": "glob",
         "allValue": null,
         "current": {
-          "text": "10.0.1.5:9100",
-          "value": "10.0.1.5:9100"
+          "text": "10.151.40.202:9100",
+          "value": "10.151.40.202:9100"
```
Comment (sterowang) [inline service-deployment/src/grafana/pai-nodeview-dashboard.json.template]: What does this IPs means?

Comment (YitongFeng) [reply service-deployment/src/grafana/pai-nodeview-dashboard.json.template]: Solved in latest commit

Comment (YitongFeng) [reply service-deployment/src/grafana/pai-nodeview-dashboard.json.template]: Solved in latest commit

File: service-deployment/src/grafana/pai-clusterview-dashboard.json.template (hunk 2 above)
Comment (YitongFeng) [reply service-deployment/src/grafana/pai-clusterview-dashboard.json.template]: Yes, Solved in latest commit

File: service-deployment/src/grafana/pai-nodeview-dashboard.json.template (hunk 3 above)
Comment (YitongFeng) [reply service-deployment/src/grafana/pai-nodeview-dashboard.json.template]: Solved in latest commit

File: service-deployment/src/grafana/pai-nodeview-dashboard.json.template (hunk 4 above)
Comment (YitongFeng) [reply service-deployment/src/grafana/pai-nodeview-dashboard.json.template]: Solved in latest commit

Comment (hwuu) [reply service-deployment/src/grafana/pai-nodeview-dashboard.json.template]: @sterowang: should we hide internal ips in code?

Comment (YitongFeng) [reply service-deployment/src/grafana/pai-nodeview-dashboard.json.template]: Sorry I forgot to modify this file. Now it is set to "" in the latest commit and works well. @sterowang @hwuu

**Pull request merged**